### DISCUSSION TITLE
「开源之史」系列之一/四：被迫的反击——捍卫开源知识财产的历程 - 开源之道

### DISCUSSION THREAD

**Discussion by giscus** [OCselected/ocselected.github.io#36] [Announcements]
# 「开源之史」系列之一/四：被迫的反击——捍卫开源知识财产的历程 - 开源之道

https://opensourceway.community/posts/history-of-open-source/02-00-ip-law-and-license-evolution-start/

**Comment by lijiangsheng1** [comment]
测试一下通知到slack 频道

**Comment by ZoomQuiet** [comment]
#是也乎,(￣▽￣)

类似案件一直在发生, 又一直在被忽略,
感觉开源之史, 值得开一个开源案件别册,
收集整理所有和 FLOSS 许可证相关的关键案件,

- 整理时间线
- 指向关键永久档案
- 分析历史意义
- 给出冲突点解释
- ...

基本上就象第谷给开普勒准备的基础数据集,
以便后人可以根据这一组凡例梳理出类似 女权崛起之路的, 开源思想复兴之路的思考...
